bundler.d/console.rb

```diff
@@ -1,4 +1,5 @@
 group :console do
   gem 'wirb', '>= 1.0', '< 3.0'
   gem 'amazing_print', '~> 1.1', :require => 'ap'
+  gem 'irb'
```

Comment:
I lean to only including this in foreman-packaging. It's a default gem and all expect Red Hat provided Rubies include those (as the name implies) by default. So IMHO it's a Red Hat packaging bug.